CMakeLists.txt

```diff
@@ -703,7 +703,7 @@ set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/grammar.c PROPERTIES
 #
 #set(PROJECT_SOURCE_LIST_C ${PROJECT_SOURCE_LIST_C} ${CMAKE_CURRENT_BINARY_DIR}/grammar.c)
 
-if(WIN32)
+if(WIN32 AND NOT MSYS AND NOT ${CMAKE_CROSSCOMPILING})
```

Comment:
So that's "cross compiling" as in "building for Windows on some flavor of UN\*X", which means "you have UN\*X tools to build `version.c` and `pcap_version.h`"?

So with MinGW on Windows *without* UN\*X tools, this will use the batch file?

Reply:
Yes. In addition, there is a cmake generator that uses UNIX tools that have been ported to windows to build the project (including a POSIX/bourne compatible shell and of course the gcc windows port). It is know as Msys. cmake stays clear of the batch file if this generator is chosen.